python/doc/examples/calibration/bayesian_calibration/plot_hierarchical_calibration_fission_gas.py

```diff
@@ -0,0 +1,577 @@
+
+_ = View(pair_plots)
+
+# %%
+# Create an enhanced pair plots grid with histograms of the marginals on the diagonal.
+
+full_grid = ot.GridLayout(pair_plots.getNbRows() + 1, pair_plots.getNbColumns() + 1)
+
+for i in range(full_grid.getNbRows()):
+    hist = ot.HistogramFactory().build(reduced_samples.getMarginal(i))
+    pdf = hist.drawPDF()
+    pdf.setLegends([""])
+    pdf.setTitle("")
+    full_grid.setGraph(i, i, pdf)
+
+for i in range(pair_plots.getNbRows()):
+    for j in range(pair_plots.getNbColumns()):
+        if len(pair_plots.getGraph(i, j).getDrawables()) > 0:
+            full_grid.setGraph(i + 1, j, pair_plots.getGraph(i, j))
+
+_ = View(full_grid)
+
+
+# %%
+# Finally superimpose contour plots of the KDE-estimated 2D marginal PDFs on the pairplots.
+
+ot.ResourceMap.SetAsBool("Contour-DefaultIsFilled", True)
+ot.ResourceMap.SetAsString("Contour-DefaultColorMap", "viridis")
+
+# sphinx_gallery_thumbnail_number = 3
+for i in range(1, full_grid.getNbRows()):
+    for j in range(i):
+        graph = full_grid.getGraph(i, j)
+        cloud = graph.getDrawable(0).getImplementation()
+        cloud.setPointStyle(".")
+        data = cloud.getData()
+        dist = ot.KernelSmoothing().build(data)
+        contour = dist.drawPDF().getDrawable(0).getImplementation()
+        contour.setLevels(np.linspace(0.0, contour.getLevels()[-1], 10))
+        graph.setDrawables([contour, cloud])
+        graph.setBoundingBox(contour.getBoundingBox())
+        full_grid.setGraph(i, j, graph)
+
+_ = View(full_grid, scatter_kw={"alpha": 0.1})
+
+
+# %%
+# Post-calibration prediction
+# ---------------------------
+#
+# Retrieve the :math:`\mu` and :math:`\sigma^2` columns in the sample.
+
+mu = samples.getMarginal([f"$\\mu$_{{{label}}}" for label in desc])
+sigma_square = samples.getMarginal([f"$\\sigma$_{{{label}}}^2" for label in desc])
+
+
+# %%
+# Build the joint distribution of the latent variables :math:`x_{\mathrm{diff}}, x_{\mathrm{crack}}`
+# obtained when :math:`\mu_{\mathrm{diff}}`, :math:`\sigma_{\mathrm{diff}}`,
+# :math:`\mu_{\mathrm{crack}}` and :math:`\sigma_{\mathrm{crack}}`
+# follow their joint posterior distribution.
+# It is estimated as a mixture of truncated :math:`\sampleSize_{\mathrm{exp}}`-dimensional normal distributions
+# corresponding to the posterior samples of the :math:`\mu_{\mathrm{diff}}`, :math:`\mu_{\mathrm{crack}}`,
+# :math:`\sigma_{\mathrm{diff}}` and :math:`\sigma_{\mathrm{crack}}` parameters.
+
+truncation_interval = ot.Interval(lbs, ubs)
+normal_collection = [
+    ot.TruncatedDistribution(ot.Normal(mean, np.sqrt(var)), truncation_interval)
+    for (mean, var) in zip(mu, sigma_square)
+]
+normal_mixture = ot.Mixture(normal_collection)
+normal_mixture.setDescription(desc)
+
+# %%
+# Build a collection of random vectors such that the distribution
+# of each is the push-forward of the marginal distribution of :math:`(x_{\mathrm{diff}}, x_{\mathrm{crack}})`
+# defined above through one of the nexp models.
```

Comment:
nexp should it be \sampleSize_{\mathrm{exp} ?

Reply:
Indeed, I have changed this.